content/applications/marketing/email_marketing/lost_leads_email.rst

```diff
@@ -0,0 +1,337 @@
+selection chosen in the third field.
+
+In the third field, use the calendar popover window to select dates, and click :guilabel:`Apply` to
+lock in the time range.
+
+.. image:: lost_leads_email/created-on.png
+   :align: center
+   :alt: A custom filter rule setting the time period to be anything before today's date.
+
+.. important::
+   When there is more than one rule applied, make sure the statement at the top of the
+   :guilabel:`Recipients` filter list reads :guilabel:`Match all of the following rules`. If it does
+   not, click on the statement, and select :guilabel:`all` from the drop-down menu (as opposed to
+   :guilabel:`any`).
+
+   .. image:: lost_leads_email/match-all.png
+      :align: center
+      :alt: The statement at the top of the filters list, with the drop-down menu open.
+
+Stage
+-----
+
+Now add :guilabel:`Stage` filter to exclude leads in the *New*, *Qualified*, and *Won* stages of
+the sales pipeline.
+
+.. note::
+   This step assumes that the *New*, *Qualified*, and *Won* stages exist in the CRM pipeline,
+   however stage names may differ from business to business. Refer to the database's actual stage
+   names in the *CRM* app's pipeline to complete this step, accordingly.
+
+Begin again by clicking :guilabel:`New Rule` and selecting :guilabel:`Stage` from the first field's\
+drop-down menu. In the second field, select the :guilabel:`is not in` operator, and in the third
+field, select the :guilabel:`New`, :guilabel:`Qualified` and :guilabel:`Won` stages to define the
+rule's parameters.
+
+When the rule is added in this way, the logic in the third field renders as OR (`|`) statements.
+
+.. image::  lost_leads_email/stage-is-in.png
+   :align: center
+   :alt: Include multiple Stages in the filtering rule, using the "is in" operator
+
+.. tip::
+   Another way to add *Stage* rules, is to do so on a one-rule-per-row basis using the
+   :guilabel:`contains` or :guilabel:`does not contain` operators, and manually typing out the
+   defining characters in each stage name. This method, however, only allows for one selection at a
+   time, which can be useful for quickly turning on/off filters in the :guilabel:`Search...` bar.
+
+
+   .. image:: lost_leads_email/stages.png
+      :align: center
+      :alt: Three filter rules requiring that the Stage does not contain New, Qualified, or Won.
+
+Lost Reason
+-----------
+
+Next, add one or more :guilabel:`Lost Reason` rules to exclude leads that should not be targeted for
+specific :doc:`lost reasons <../../sales/crm/pipeline/lost_opportunities>`.
+
+Create another :guilabel:`New Rule`, and in the rule's first field, select :guilabel:`Lost Reason`
+from the drop-down menu. For the operator, choose either :guilabel:`is not in` or :guilabel:`does
+not contain` from the drop-down menu. With either selection, use the third field to enter a lost
+reason (or multiple lost reasons, depending on your operator choice) to include in the rule.
```

Comment:
```suggestion
To do that, click :guilabel:`New Rule` once again. Then, in the rule's first field, select
:guilabel:`Lost Reason` from the drop-down menu.

For the operator in the second field, choose either :guilabel:`is not in` or :guilabel:`does not
contain` from the drop-down menu.

With either of those selections in the second field, use the third field to enter a lost reason (or
multiple lost reasons, depending on the operator choice) to include in the rule.
```

Reply:
Added transitional language.
I don't, however, like to break up paragraphs that are a part of the same breath of instructions.